autopilot/contracts.go

```diff
@@ -1,27 +1,413 @@
 package autopilot
 
 import (
+	"math"
+	"time"
+
+	"gitlab.com/NebulousLabs/errors"
+	"go.sia.tech/renterd/bus"
 	"go.sia.tech/renterd/internal/consensus"
+	rhpv2 "go.sia.tech/renterd/rhp/v2"
 	"go.sia.tech/renterd/worker"
 	"go.sia.tech/siad/types"
 	"golang.org/x/crypto/blake2b"
 )
 
-func (ap *Autopilot) currentHeight() uint64 {
-	return ap.store.Tip().Height
+const (
+	// contractSetName defines the name of the default contract set
+	contractSetName = "autopilot"
+
+	// contractLoopInterval defines the interval with which we run the contract loop
+	contractLoopInterval = 15 * time.Minute
+
+	// estimatedFileContractTransactionSetSize is the estimated blockchain size
+	// of a transaction set between a renter and a host that contains a file
+	// contract.
+	estimatedFileContractTransactionSetSize = 2048
+)
+
+func logErr(err error) {} // TODO
```

Comment:
Wouldn't we want to use something like logrus and then derive a logger for the autopilot that we pass in through a constructor? Rather than having a global logging method?

Reply:
Oh yeah for sure. It's just a placeholder for me (personally) to not care about errors, error handling and logging for the time being. Logging is done in follow up PR.